web/src/components/eventslist-component/EventFilters.js

```diff
@@ -0,0 +1,77 @@
+import React, {Component} from 'react';
+import Cards from './EventCards';
+import {
+  ToggleButton,
+  ToggleButtonGroup,
+  Jumbotron
+} from 'react-bootstrap';
+
+
+class EventFilters extends Component {
+  constructor(props) {
+    // get area selection from props
+    super(props);
+
+    // save the arrays of filter names and filter toggle(selected or not) in state
+    // Todo: mapping filterNames string-integer(enum) after db builiding complete.
+    const filterNames = ['BookClub', 'Movie', 'Wine', 'SciFi', 'Sport'];
+    const filterToggles = [];
+
+    for (let j = 0; j < filterNames.length; j++)
```

Comment:
Same here. Using random letter is not a good naming convention. Try to come up with something more meaningful.